dev/ChangeTracker/EnvironmentVariableChangeTracker.h

```diff
@@ -41,52 +41,30 @@ namespace winrt::Microsoft::ProjectReunion::implementation
 
         wil::unique_hkey GetKeyForTrackingChange()
         {
-            std::wstringstream subKeyStream;
-            subKeyStream << L"Software\\ChangeTracker\\";
-            subKeyStream << KeyName();
-            subKeyStream << L"\\";
-            subKeyStream << m_PackageFullName;
-            subKeyStream << "\\";
-            subKeyStream << m_Key;
-            subKeyStream << "\\";
+            HKEY topLevelKey;
```

Comment:
Does Project Reunion have a case for, or against these kind of if statements?

Reply:
Ternary operators are fine (no restrictive guidance like "never use if without {}")

Simple ternary operations like here simplify variable initialization e.g.
```
int foo{};
if (x)
{
    foo = y;
}
else
{
    foo = z;
}
```

vs

```
int foo{ x ? y : z };
```

This also enables `auto foo{ x ? y : z };` when auto makes sense. The former precludes auto since `auto foo{};` isn't viable